x/mongo/driver/topology/topology.go

```diff
@@ -173,6 +174,69 @@ func logTopologyMessage(topo *Topology, msg string, keysAndValues ...interface{}
 		}, keysAndValues...)...)
 }
 
+func mustLogServerSelectionMessage(topo *Topology, level logger.Level) bool {
```

Comment:
Optional: Adding "Message" to the end of these functions makes them overly wordy and doesn't add much information. Consider removing the word "Message" from the function names.

That recommendation applies to all of the `*logServerSelection*` functions.